src/ngraph/runtime/interpreter/int_backend.cpp

```diff
@@ -164,35 +171,37 @@ bool runtime::interpreter::INTBackend::call(shared_ptr<Function> function,
 
         // get op type
         element::Type type;
-        if (dynamic_pointer_cast<op::util::BinaryElementwiseComparison>(op) ||
-            dynamic_pointer_cast<op::Select>(op))
+        switch (type_id)
         {
+        case OP_TYPEID::Convert_TYPEID:
```

Comment:
`OP_TYPEID::x_TYPEID` seems redundant, and with CamelCase we shouldn't be hitting reserved words, so how about `OP_TYPEID::x`?

Reply:
done